charts/airbyte/values.yaml

```diff
@@ -0,0 +1,380 @@
+  ## ref: https://kubernetes.io/docs/tasks/configure-pod-container/configure-liveness-readiness-probes/#configure-probes
+  ## @param server.livenessProbe.enabled Enable livenessProbe on the server
+  ## @param server.livenessProbe.initialDelaySeconds Initial delay seconds for livenessProbe
+  ## @param server.livenessProbe.periodSeconds Period seconds for livenessProbe
+  ## @param server.livenessProbe.timeoutSeconds Timeout seconds for livenessProbe
+  ## @param server.livenessProbe.failureThreshold Failure threshold for livenessProbe
+  ## @param server.livenessProbe.successThreshold Success threshold for livenessProbe
+  ##
+  livenessProbe:
+    enabled: true
+    initialDelaySeconds: 30
+    periodSeconds: 10
+    timeoutSeconds: 1
+    failureThreshold: 3
+    successThreshold: 1
+
+  ## @param server.readinessProbe.enabled Enable readinessProbe on the server
+  ## @param server.readinessProbe.initialDelaySeconds Initial delay seconds for readinessProbe
+  ## @param server.readinessProbe.periodSeconds Period seconds for readinessProbe
+  ## @param server.readinessProbe.timeoutSeconds Timeout seconds for readinessProbe
+  ## @param server.readinessProbe.failureThreshold Failure threshold for readinessProbe
+  ## @param server.readinessProbe.successThreshold Success threshold for readinessProbe
+  ##
+  readinessProbe:
+    enabled: true
+    initialDelaySeconds: 10
+    periodSeconds: 10
+    timeoutSeconds: 1
+    failureThreshold: 3
+    successThreshold: 1
+
+  ## Server app resource requests and limits
+  ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
+  ## We usually recommend not to specify default resources and to leave this as a conscious
+  ## choice for the user. This also increases chances charts run on environments with little
+  ## resources, such as Minikube. If you do want to specify resources, uncomment the following
+  ## lines, adjust them as necessary, and remove the curly braces after 'resources:'.
+  ## @param server.resources.limits [object] The resources limits for the server container
+  ## @param server.resources.requests [object] The requested resources for the server container
+  resources:
+    ## Example:
+    ## limits:
+    ##    cpu: 200m
+    ##    memory: 1Gi
+    limits: {}
+    ## Examples:
+    ## requests:
+    ##    memory: 256Mi
+    ##    cpu: 250m
+    requests: {}
+
+  ## @param server.service.type The service type to use for the API server
+  ## @param server.service.port The service port to expose the API server on
+  service:
+    type: ClusterIP
+    port: 8001
+
+  ## @param server.persistence.accessMode The access mode for the airbyte server pvc
+  ## @param server.persistence.storageClass The storage class to use for the airbyte server pvc
+  ## @param server.persistence.size The size of the pvc to use for the airbyte server pvc
+  persistence:
+    accessMode: ReadWriteOnce
+    storageClass: standard
```

Comment:
I think you mentioned that you're planning on changing this away from `standard`?

Reply:
@jrhizor, yes. This was a mistake on my part. I'll have a patch out for it shortly. In our EKS cluster we have a storage class called `standard` and that is also the name of the storage class in the kind cluster I was using for local dev so  I didn't hit this issue sooner.

```
kubectl get storageclass
NAME                 PROVISIONER             RECLAIMPOLICY   VOLUMEBINDINGMODE      ALLOWVOLUMEEXPANSION   AGE
standard (default)   rancher.io/local-path   Delete          WaitForFirstConsumer   false                  3m56s
```

Reply:
This should be fixed in https://github.com/airbytehq/airbyte/pull/5891/commits/2df0ec4e75a563e6954e51ef6002bb82c4881e1a